client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/useInvestigatedPersonInfo.ts

```diff
@@ -46,8 +46,13 @@ const useInvestigatedPersonInfo = (): InvestigatedPersonInfoOutcome => {
         timeout(1900).then(()=> history.push(landingPageRoute));
     };
 
+    const getPersonAge = (birthDate: Date) => {
+        return new Date().getFullYear() - new Date(birthDate).getFullYear()
```

Comment:
not exactly a true condition 🤔

Reply:
מה?

Reply:
I think she means that the calc isn't really accurate, becaue if it's 06/09/2020 and you were born at 11/11/1999 than it will be 21 even thought you are still 20

Reply:
got it. Tx